src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,88 @@
+package lotto.view;
+
+import java.text.NumberFormat;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import lotto.Lotto;
+import lotto.domain.Rank;
+
+public class OutputView {
+  private static final NumberFormat NUMBER_FORMAT = NumberFormat.getInstance(Locale.KOREA);
+
+  public static void printPurchaseResult(List<Lotto> lottos) {
+    System.out.println();
+    System.out.println(lottos.size() + "개를 구매했습니다.");
```

Comment:
```suggestion
    System.out.println("\n"+ lottos.size() + "개를 구매했습니다.");
```
개행문자를 추가하면 코드를 더 간결하게 표현할 수 있을 거 같아요.